### DISCUSSION TITLE
去中心化是否真的去中心?

### DISCUSSION THREAD

**Discussion by yuhao112** [1101-GS4719-C/Python-Community-GS4719-C#549] [科技新知]
https://medium.com/coinmonks/as-metamask-and-opensea-abide-by-us-sanctions-the-russian-ban-is-in-the-spotlight-67611df121f8
俄烏戰爭期間，交易nft 的最大交易所依照美國的法規條例制裁來自俄羅斯的用戶。先不論俄羅斯發動戰爭是否正確，但對於 opensea 制裁用戶這塊存在著很大的疑慮:標榜去中心化的場所，卻因為政治立場而變化，是否真如其所說真的去中心?然而文章中opensea依舊聲稱無政治立場...。就我認為在任何國家下完全去中心化那是不可能的，很難是在沒有拘束下去交易的，只能說區塊鏈帶來的所謂的去中心化對我來說的概念就是背後少了些單位能內部操作的空間而已。

**Comment by ghost** [comment]
去中心化的概念應該理解成伺服器去中心化，但本質還是建立在所有使用者上 既然是人，有人為操作空間也許不用那麼意外@@

**Comment by swerfg03joe** [comment]
新的名詞，新的遊戲規則，去中心化其實什麼都沒有去除，講難聽點就是詐騙，NFT本質上就是一種龐氏騙局，一開始一堆人往裏頭衝，確實有人賺到財富，不過是一個願打一個願挨而已，當後繼無力時，裡面來不及出來的人全都會血本無歸。因NFT本身並不具任何價值

**Comment by harrylee1971** [comment]
我贊同上面那位同學的想法，NFT就是一個大騙局，但或許去中心化比起原本的各國銀行是較為優秀的一點

**Comment by woohsin** [comment]
你提出的觀點很值得深思。事實上，雖然區塊鏈技術被認為是一種去中心化的技術，但實際上在現實應用中，仍然難以實現完全的去中心化。就像你所提到的，即使是在區塊鏈上運行的交易所，也可能受到政治、法律和其他因素的影響，導致對特定用戶或特定國家實施制裁或限制。

這表明了區塊鏈技術在實際應用中所面臨的一些挑戰，包括法律法規的規範、政治立場的影響等。雖然區塊鏈技術本身可能是去中心化的，但在實際應用中，仍然難以避免受到外部因素的影響。

這也提醒我們，當探討區塊鏈技術時，我們不能只看到其技術本身，還需要考慮到其在現實應用中所面臨的各種挑戰和限制。因此，我們在利用區塊鏈技術時，需要更加理性和全面地看待其優缺點，並根據實際情況做出適當的應對。

**Comment by jylin34** [comment]
去中心化技術在2008年由中本聰提出，其核心理念是通過數學和密碼學的方法取代人工監管，以數學取代人性。比特幣自推出以來已穩定運行16年，間接證明了去中心化技術的可行性。然而，現今世界面臨的挑戰是如何建立真正去中心化制度來取代、避免各種中心化機構監管。

**Comment by NcuJerryKao** [comment]
OpenSea在俄烏戰爭期間，依照美國的法規制裁來自俄羅斯的用戶，這引發了對其去中心化聲明的質疑。這種行為顯示了在國際政治壓力下，標榜去中心化的平台也難以完全保持中立。儘管OpenSea聲稱無政治立場，但實際操作上卻顯示出其必須遵從某些政治和法律框架。這暴露了去中心化理想與現實的矛盾：在任何國家下，完全的去中心化是不可能的，因為一旦涉及國際制裁和法律責任，平台仍需作出相應調整。

我認為在追求去中心化的同時，應該更現實地看待其局限性。去中心化的概念在區塊鏈技術中主要體現在減少單一控制點，增強系統的透明度和安全性，但這並不意味著完全脫離政府和法律的監管。平臺應該在技術上儘可能地保障用戶的自由交易，同時也要遵守各國法律，這樣才能在合法合規的基礎上發展。

此外，用戶在參與這類平台時，也需要提高自身的法律和風險意識，了解可能面臨的政策變化和法律風險。這樣，不僅能更好地保護自己的權益，還能在面對政策變動時迅速做出應對。總之，去中心化是一個方向，但並非全能解決方案，平衡技術創新與法律合規，是未來區塊鏈和NFT平台發展的關鍵。

**Comment by alexender0104** [comment]
去中心化是一個複雜的概念，涵蓋了從技術到社會組織的不同層面。在理論上，去中心化的目標是消除或減少中央權力和控制，使得系統或組織更加公平和分散。但在實際應用中，完全去中心化是很難實現的，我認為因為個人為因素存在，去中心化系統仍然需要人類的設計、維護和管理。設計者和開發者的偏見、決策和行動會影響系統的運作方式，這在某種程度上重新引入了中心化的元素。

**Comment by Shang621** [comment]
去中心化是否真的去中心，是一個值得深入探討的問題。雖然去中心化技術和平台聲稱能夠消除單一中心控制，但實際上仍可能存在隱藏的中心化因素。例如，某些去中心化平台可能受到少數核心開發人員或組織的影響，導致其仍然受制於一個中心化實體。此外，使用去中心化技術的過程中，仍然可能需要依賴中心化服務或平台，例如加密貨幣交易所。因此，雖然去中心化帶來了某些優勢，但我們仍應保持警覺，避免盲目相信其完全去中心化的宣稱，並在使用時保持理性和審慎。

**Comment by sandyho0307** [comment]
對我來說，區塊鏈和去中心化的真正意義並不是完全擺脫所有的監管和控制，而是減少中間環節和內部操作的空間。這意味著交易和運營變得更加透明和公平，但並不等於可以完全無視法律和規範。

**Comment by 710388** [comment]
是吧，我記得我在網路上看過比特幣的演算法，非常的高明，真的是藝術品，感覺照他的邏輯組合起來應該是去中心化的
